mod_dynamic_spawns/classes/party.nut

```diff
@@ -55,6 +77,17 @@
 			}
 		}
 
+		this.__SubParties = [];
```

Comment:
I'm confused that you call this field `__SubParties` when it is created purely from categories.
What about `__Categories` instead?

Reply:
`Categories` is a frontend variable used by "designers" to design parties in code. Whereas in fact they are actual party objects. So the private variable used here is called `SubParties` to make it clear in code that these are party objects and I prefer this.